test/verify/check-networking-basic

```diff
@@ -165,6 +171,11 @@ class TestNetworkingBasic(NetworkCase):
         assert_running()
         wait(lambda: m.execute("systemctl is-enabled NetworkManager"))
 
+        # remove the DBus service, test 'no-found' notification
+        self.restore_file('/usr/lib/systemd/system/NetworkManager.service')
```

Comment:
If this is a non-destructive test, we need to make sure that NM is started again, and that systemd is reloaded after the file is put back, no?

Reply:
I guess you 're right, but I don't see any failing tests actually

Reply:
ah there is self.addCleanup(m.execute, "systemctl enable --now NetworkManager") in the start of the test

Reply:
Correct, resolving.